src/components/Landing/Landing.jsx

```diff
@@ -1,20 +1,28 @@
 import React from "react";
+import RENDER from "SHARED/renderUtils.jsx";
+import { s, data } from "./";
 
 export default function Landing(){
+
+	const {
+		heading,
+		subheading,
+		body: bodyData
+	} = data;
+
+	const body = RENDER.body(bodyData, "landing", s.paragraph); //scope name for key, classname
```

Comment:
now that stuff's getting more complication with this function, rather than have
```
renderBody(data, scope, className){ ... }
```
as our definition, can we instead do:
```
renderBody(data, options){
    const { scope, className } = options;
    ...
}
```
That way  we can use it like:
```
const body = RENDER.body(bodyData, { scope: "landing", className: s.paragraph });
```
or
```
const body = RENDER.body(bodyData, { className: s.paragraph });
```
and we don't need to worry about passing in `null` or `undefined` when we want to skip parameters.  This kinda format is used to distinguish between essential arguments and optional ones (with the optional ones existing in the `{}`)